pkg/trait/camel.go

```diff
@@ -200,10 +207,10 @@ func (t *camelTrait) computeConfigMaps(e *Environment) []ctrl.Object {
 					v1.IntegrationLabel: e.Integration.Name,
 				},
 				Annotations: map[string]string{
-					"camel.apache.org/source.language":    string(s.InferLanguage()),
-					"camel.apache.org/source.loader":      s.Loader,
-					"camel.apache.org/source.name":        s.Name,
-					"camel.apache.org/source.compression": strconv.FormatBool(s.Compression),
+					camelSourceLanguageAnnotation:    string(s.InferLanguage()),
+					camelSourceLoaderAnnotation:      s.Loader,
+					camelSourceNameAnnotation:        s.Name,
+					camelSourceCompressionAnnotation: strconv.FormatBool(s.Compression),
```

Comment:
although replacing string with consts is desirable, since it does not seems to be strictly related with goal of PR, can this be moved to a dedicate commit at least ?

Reply:
it absolutely is but it makes understanding the specific PR goal more complex as it is hard to see a relation

Reply:
I agree. I'll take care of it, no worries.

Reply:
done